src/databricks/labs/ucx/queries/views/reconciliation_results.sql

```diff
@@ -0,0 +1,26 @@
+with flattened as (
```

Comment:
```suggestion
WITH flattened AS (
```
styleguide: can we use consistently use ALL CAPS for SQL keywords and functions?